eng/create-workload-drops.ps1

```diff
@@ -55,12 +55,11 @@ Get-ChildItem -Path $workloadDropPath -Directory | ForEach-Object {
     $shortName = "$($Matches.short)"
     # Remove the '.' from 'pre.components'
     $dropType = $Matches.type.Replace('.', '')
-    $dropUrl = "https://vsdrop.microsoft.com/file/v1/$vsDropName;$assemblyName.vsman"
 
-    Write-Host "##vso[task.setvariable variable=$($shortName)_$($dropType)_name]$vsDropName"
-    Write-Host "##vso[task.setvariable variable=$($shortName)_$($dropType)_dir]$dropDir"
-    Write-Host "##vso[task.setvariable variable=$($shortName)_$($dropType)_url]$dropUrl"
```

Comment:
The `_url` variable was unused. It was created during insertion development but no longer had a purpose.